sessionctx/varsutil/varsutil.go

```diff
@@ -30,50 +30,68 @@ import (
 // Returns error if there is no such variable.
 func GetSessionSystemVar(s *variable.SessionVars, key string) (string, error) {
 	key = strings.ToLower(key)
+	gVal, ok, err := GetSessionOnlySysVars(s, key)
+	if err != nil || ok {
+		return gVal, errors.Trace(err)
+	}
+	gVal, err = s.GlobalVarsAccessor.GetGlobalSysVar(key)
+	if err != nil {
+		return "", errors.Trace(err)
+	}
+	s.Systems[key] = gVal
```

Comment:
Should we remove this?

Reply:
This is useful, CI failed since the two commits use an interface which is defined by other commits.
I'll take a look.